examples/helloworld/greeter_client/main.go

```diff
@@ -47,9 +49,19 @@ const (
 	defaultName = "world"
 )
 
+var withInsecureTLS = flag.Bool("insecure_tls", false, "Use Insecure TLS; suitable for hitting the greeter_server in -use_http mode")
```

Comment:
Oh, you mean I should use the root CA option? I remember seeing that somewhere.

I could do that.